### PR TITLE
[ticket/13930] Check for correct spacing between keyword & parenthesis

### PR DESCRIPTION
**Pull request by marc1706**
The codesniffer should check for a correct spacing between keywords like
if or foreach and its opening parenthesis.

Ticket: https://tracker.phpbb.com/browse/PHPBB3-13930

PHPBB3-13930

### CODE REVIEW

File: build/code_sniffer/ruleset-php-strict.xml [hunk 1]
```diff
@@ -45,4 +45,7 @@
  <!-- There MUST NOT be unused use statements. -->
  <rule ref="./phpbb/Sniffs/Namespaces/UnusedUseSniff.php" />
 
+ <!-- There MUST be one space between control structure and opening parenthesis -->
+ <rule ref="./phpbb/Sniffs/ControlStructures/OpeningParenthesisSniff.php" />
```
Comment (bantu) [inline build/code_sniffer/ruleset-php-strict.xml]: Did you try putting this into legacy?

Comment (bantu) [inline build/code_sniffer/ruleset-php-strict.xml]: Do things explode?

Comment (marc1706) [inline build/code_sniffer/ruleset-php-strict.xml]: Haven't tried yet

Comment (marc1706) [inline build/code_sniffer/ruleset-php-strict.xml]: Testing now :)

**Pull request merged**